services/courses/topicOnCourse/createTopicOnCourse.ts

```diff
@@ -0,0 +1,47 @@
+export const createTopicOnCourse = async (
+  nameTopic: string,
+  userId: string,
+  idCourse: string
+) => {
+  try {
+    console.log("data SERVISES", { nameTopic, userId, idCourse });
```

Comment:
review. On this, is SERVICES, not SERVI"S"ES

Reply:
Line deleted